src/mon/OSDMonitor.cc

```diff
@@ -12829,6 +12829,13 @@ bool OSDMonitor::prepare_command_impl(MonOpRequestRef op,
 
     string poolstr;
     cmd_getval(cmdmap, "pool", poolstr);
+    bool confirm = false;
```

Comment:
why do add the 'yes I do' requirement to this command? and if needed - why not in a separate commit?

Reply:
`mgr_module.py` uses cli commands to create it's pool. Extra flag must be used (when creating `.mgr` pool) in order to differentiate between our internal creations and the rest.

Reply:
Comment was added for clarification.